apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -699,4 +707,4 @@ int smartfs_powercut_main(int argc, char *argv[])
 	}
 	
 	return OK;
-}
+
```

Comment:
Need the closing bracket. Change needs to be undone.

Reply:
My mistake, updated, thanks.